### PR TITLE
Api calls

### PR DESCRIPTION
**Pull request by pgoulding**
API calls and adding components

### CODE REVIEW

File: src/ApiFetch.js [hunk 1]
```diff
@@ -1,21 +1,62 @@
...
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(user)
+    }
+    const userSignInURL = 'http://localhost:3001/api/users/'
+    const response = await fetch(userSignInURL, options)
+    const parsed = await response.json()
+    return parsed
+  } catch (error) {
+    throw Error('we have encoutnered an error: ', error)
```
Comment (baldm0mma) [inline src/ApiFetch.js:40]: Do we want to more specific with this error?

File: src/ApiFetch.js [hunk 2]
```diff
@@ -1,21 +1,62 @@
...
+      headers: {
+        'Content-Type':'application/json'
+      },
+      body: JSON.stringify(newAccount)
+    }
+    const newUserUrl = 'http://localhost:3001/api/users/new'
+    const response = await fetch(newUserUrl, options)
+    return response
+  } catch(error) {
+    throw Error('we have encoutnered an error: ', error)
```
Comment (baldm0mma) [inline src/ApiFetch.js:57]: Same with this?

File: src/components/Card/Card.js [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+import './Card.css'
 const Card = ({ movieInfo }) => {
   const { title, poster_path } = movieInfo
 
   const imageUrl = `http://image.tmdb.org/t/p/w300/${poster_path}`
   return (
     <article className="card">
       <h3>{ title }</h3>
-      <img src={imageUrl}/>
+      <img alt={ title && ' movie poster' } src={ imageUrl }/>
```
Comment (baldm0mma) [inline src/components/Card/Card.js:10]: Like this!

File: src/containers/UserMenu/UserMenu.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+      const results = await user.data
+      this.props.signIn(results)
+    } catch (error) {
+      this.setState({error})
+    }
+    // this.props.createAccount({thisstae})
+  }
+
+
+  render() {
```
Comment (baldm0mma) [inline src/containers/UserMenu/UserMenu.js:35]: Should we split this up into a couple of components?

**Review by baldm0mma — APPROVED** [review]

File: src/containers/UserMenu/UserMenu.js (hunk 4 above)
Comment (pgoulding) [reply src/containers/UserMenu/UserMenu.js:35]: Hmm, good idea.

File: src/ApiFetch.js (hunk 1 above)
Comment (pgoulding) [reply src/ApiFetch.js:40]: Good idea, let's talk about this later!

**Pull request merged**